content/developer/reference/frontend/javascript_modules.rst

```diff
@@ -51,13 +51,23 @@ execute them precisely in that order).
 Native Javascript Modules
 =========================
 
-Most new Odoo javascript code should use the native javascript module system. This
-is simpler, and brings the benefits of a better developer experience with a better
-integration with the IDE.
+Odoo javascript code uses the native javascript module system. This is simpler, and
+brings the benefits of a better developer experience with a better integration with the IDE.
 
-There is a very important point to know: Odoo needs to know which files
-should be translated into :ref:`Odoo modules <frontend/modules/odoo_module>` and which
-files should not be translated. This is an opt-in system: Odoo will look at the
+There is a very important point to know: by default Odoo transpiles files under
+`/static/src` and `/static/tests` into :ref:`Odoo modules <frontend/modules/odoo_module>`.
+This is an opt-out system, it's possible to tell the transpiler to ignore the file.
+
+.. code-block:: javascript
+
+  /** @odoo-module ignore **/
+  (function () {
+    const sum = (a, b) => a + b;
+    console.log(sum(1, 2));
+  )();
+
+In other folders Odoo needs to know which files should be transpiled and which
+files should not be transpiled. This is an opt-in system: Odoo will look at the
 first line of a JS file and check if it contains the string *@odoo-module*. If so, it will
 automatically be converted to an Odoo module.
```

Comment:
```suggestion
In other folders, files aren't transpiled default, it is opt-in. Odoo will look at the
first line of a JS file and check if it contains the string *@odoo-module*. If so, it will
automatically be converted to an Odoo module.
```